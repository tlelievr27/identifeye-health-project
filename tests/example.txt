ADD PATIENT 123 JOHN DOE
ADD PATIENT 321 JOE SCMOE
ADD PATIENT 321 JOHN SNOW
ADD PATIENT 789 JANE CROW
ADD EXAM 321 444
ADD EXAM 789 445
ADD EXAM 789 554
DEL PATIENT 321