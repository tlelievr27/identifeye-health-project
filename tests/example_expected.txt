Name: JOHN DOE, Id: 123, Exam Count: 0
Name: JANE CROW, Id: 789, Exam Count: 2